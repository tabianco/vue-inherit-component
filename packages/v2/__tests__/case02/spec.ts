import { mount } from '@vue/test-utils'

import WrappedComponent from './Wrapped.vue'

describe('<template> w/ default slot', () => {
  it('renders correctly', () => {
    const wrapper = mount(WrappedComponent)

    expect(wrapper.element).toMatchSnapshot()
  })

  it('should re-render', async () => {
    const wrapper = mount(WrappedComponent)

    await wrapper.setProps({
      value: 1
    })

    expect(wrapper.element).toMatchSnapshot()
  })
})
