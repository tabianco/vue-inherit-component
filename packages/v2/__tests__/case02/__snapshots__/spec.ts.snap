// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<template> w/ default slot renders correctly 1`] = `
<div
  class="inherited-component"
>
  <div>
    
    Child 1
  
  </div>
  <div>
    
    Child 2
  
  </div>
  <div>
    
    Child 3
  
  </div>
</div>
`;

exports[`<template> w/ default slot should re-render 1`] = `
<div
  class="inherited-component"
>
  <div>
    
    Child 1
  
  </div>
</div>
`;
